modules/atone.h

```diff
@@ -0,0 +1,75 @@
+// # ATone
+//
+// A first-order recursive high-pass filter with variable frequency response.
+//
+// ## Credits  
+//   Csound / Soundpipe
```

Comment:
I'd flesh this out a bit to include the original authors, and also yourself for porting it:
i.e.
```
## Credits

* Original Author(s): Barry Vercoe, John FFitch, Gabriel Maldonado
* Year: 1991
* Original Location: Csound -- OOps/ugens5.c
* Ported from soundpipe by <your name>, May 2020
```

Or similar (you don't have to put your full name or anything, but you did the porting so you should get some credit!)

We don't really have a format down for things with such a long history, so you can just bullet list them like that for now.